File: app/src/main/java/com/robillo/oreomusicplayer/adapters/holders/SongHolder.java
package com.robillo.oreomusicplayer.adapters.holders;

import android.support.v7.widget.CardView;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.robillo.oreomusicplayer.R;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * Created by robinkamboj on 11/09/17.
 */

public class SongHolder extends RecyclerView.ViewHolder{

    @BindView(R.id.title)
    public TextView title;
    @BindView(R.id.artist_duration)
    public TextView artistDuration;
    @BindView(R.id.song_card)
    public CardView songCard;


    public SongHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/com/robillo/dancingplayer/views/activities/main/song_list_frag/adapters/SongHolder.kt
package com.robillo.dancingplayer.views.activities.main.song_list_frag.adapters

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageButton
import kotlinx.android.synthetic.main.row_song.view.*

class SongHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    var parentView: View = itemView
    var moreButton: ImageButton = itemView.more
    var linearLayout = itemView.linear_layout
    var songCard = itemView.song_card
    var mTitle = itemView.title
    var artistDuration = itemView.artist_duration
    var albumArt = itemView.album_art
}
